client/service/src/lib.rs

```diff
@@ -326,8 +314,7 @@ fn start_rpc_servers<R>(
 where
 	R: Fn(sc_rpc::DenyUnsafe) -> Result<RpcModule<()>, Error>,
 {
-	let (max_request_size, ws_max_response_size, http_max_response_size) =
-		legacy_cli_parsing(config);
+	let (max_request_size, ws_max_response_size, _) = legacy_cli_parsing(config);
```

Comment:
Nit: Would it make sense to extract `http_max_response_size` here and then create two separate `sc_rpc_server::Config`?

It seems that both servers do receive a `ws_max_response_size` now

Reply:
nice catch, yeah you are right